src/core/a-entity.js

```diff
@@ -127,6 +129,28 @@ var proto = Object.create(ANode.prototype, {
     }
   },
 
+  getObject3D: {
+    value: function (type, Constructor) {
+      var object3D = this.objects3D[type];
+      if (!object3D && Constructor) {
+        object3D = this.objects3D[type] = new Constructor();
```

Comment:
Yep, we were talking about this before, the problem is when you want to create a ```light``` for example, it could be a ```ambient```, ```directional```, ```spot```,... 
so it's better to leave this step to the user.